app/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_experiment_clone_serializer.py

```diff
@@ -77,3 +79,43 @@ def test_invalid_rollout_branch_slug(self):
         )
         self.assertFalse(serializer.is_valid())
         self.assertIn("rollout_branch_slug", serializer.errors)
+
+    def test_locales_only_clone_for_desktop_application(self):
+        desktop_application = NimbusExperiment.Application.DESKTOP
+        locale = LocaleFactory.create()
+
+        parent = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.ENDING_APPROVE_APPROVE,
+            application=desktop_application,
+            locales=[locale.id],
+        )
+        serializer = NimbusExperimentCloneSerializer(
+            data={
+                "parent_slug": parent.slug,
+                "name": "New Name",
+            },
+            context={"user": parent.owner},
+        )
+        self.assertTrue(serializer.is_valid())
+        child = serializer.save()
+        self.assertEqual(list(child.locales.all()), [locale])
+
+    def test_locales_not_clone_for_mobile_application(self):
```

Comment:
Yep, could've used `parameterize` to test all the mobile applications here, but I don't think it's super critical, just something to remember whenever want something to be true for all mobile applications.